frontend/vite.config.js

```diff
@@ -21,3 +23,7 @@ module.exports = {
     // },
   },
 };
+
```

Comment:
Let's add a comment here. BTW, I am curious how did you figure this out?

Reply:
A guy from Gitpod got back to me the other day, it is actually his credit lol.
They seems to conduct some network analysis and locates the problem.

Reply:
Learn something new!